src/gui/windows/multiplayer.zig

```diff
@@ -56,14 +56,17 @@ fn join(_: usize) void {
 		ipAddress = "";
 	}
 	if(connection) |_connection| {
+		if (ipAddressEntry.currentString.items.len == 0) {
```

Comment:
I'm sure there is whole host of related errors from malformed IP addresses, like probably all UnknownHost and ConnectionRefused errors.

Do you think you can figure out how to handle all of them, preferably from catching the source error?